tools/patches/011-component-color-contrast.patch

```diff
@@ -0,0 +1,175 @@
+diff --git a/inst/lib/bs-sass/assets/stylesheets/bootstrap/_variables.scss b/inst/lib/bs-sass/assets/stylesheets/bootstrap/_variables.scss
+index 9ed7cb4..73636b7 100644
+--- a/inst/lib/bs-sass/assets/stylesheets/bootstrap/_variables.scss
++++ b/inst/lib/bs-sass/assets/stylesheets/bootstrap/_variables.scss
+@@ -124,10 +124,10 @@ $border-radius-base:        4px !default;
+ $border-radius-large:       6px !default;
+ $border-radius-small:       3px !default;
+ 
+-//** Global color for active items (e.g., navs or dropdowns).
+-$component-active-color:    #fff !default;
+ //** Global background color for active items (e.g., navs or dropdowns).
+ $component-active-bg:       $brand-primary !default;
++//** Global color for active items (e.g., navs or dropdowns).
++$component-active-color:    color-contrast($component-active-bg) !default;
```

Comment:
```suggestion
+$component-active-color:    color-contrast($component-active-bg, #fff) !default;
```

Reply:
Doing this properly would be complicated for no clear benefit (`$white` is considered first in the BS3 case anyway)